Cargo.lock

```diff
@@ -3547,15 +3355,11 @@ dependencies = [
 
 [[package]]
 name = "quote"
-version = "1.0.10"
+version = "1.0.18"
```

Comment:
This is required by enum-iterator-derive v0.8.1 by enum-iterator v0.8.1 by solana-program-runtime v1.11.0

Reply:
Sounds good, this impacts proc-macro2 as well